subsys/bluetooth/controller/Kconfig

```diff
@@ -18,6 +18,13 @@ config BLECTRL_SLAVE_COUNT
 	  NOTE: the number of master roles is defined as
 	  BT_MAX_CONN - BLECTRL_SLAVE_COUNT
 
+config BLECTRL_MAX_CONN_EVENT_LEN
+	int "Max connection event length [us]"
+	default 7500
+	help
+	  Maximum connection event length is set as part of BLE controller
```

Comment:
Should have some more explanation to what this is.

From SD header file (adapted for zephyr):
The time set aside for connections on every connection interval in microseconds.
The event length and the connection interval are the primary parameters for setting the throughput of a connection.

Reply:
Added the description